backend/www/manage_event.php

```diff
@@ -41,6 +41,20 @@ function validate_json_request($data) {
     return $validator->validate(TRUE, TRUE);
 }
 
+function validate_email($email) {
+    // lowercase provided email before checking against ban list
+    $email = strtolower($email);
```

Comment:
should also `s/\+.*@/@/` so users can't evade the banlist with `me+something@whatever.com` variants of their e-mail.

Reply:
Good idea, updated the PR with a check for plus tags. If one or more tags are the email address, they're stripped before checking the address against the ban list: `invalid+sneaky@example.com`, `invalid+sneaky+sneakier@example.com`, etc